packages/router/src/grpc/view-protocol-server/transaction-planner.ts

```diff
@@ -35,16 +35,15 @@ export const transactionPlanner: Impl['transactionPlanner'] = async (req, ctx) =
     viewServer: { fullViewingKey },
     querier,
   } = await services.getWalletServices();
-  const chainParams = await services.querier.app.chainParams();
   const fmdParams = await indexedDb.getFmdParams();
-  if (!fmdParams) throw new ConnectError('Fmd Params not in indexeddb', Code.FailedPrecondition);
+  if (!fmdParams) throw new ConnectError('FmdParameters not available', Code.FailedPrecondition);
```

Comment:
Are there benefits for throwing a `ConnectError` versus a normal `Error`? If so, should this be the standard in all our rpc methods?

Reply:
- we can specify a meaningful error code
- we can deliberately attach information or context that serializes
- there are some wrappers to catch and rethrow but generally yes